### ISSUE TITLE
Footnotes don't work on the website

### ISSUE THREAD

**Issue by michaelpj** [cardano-foundation/CIPs#435]
They do work in Github markdown, so this is confusing for submitters. Certainly my CIPs have had lots of footnotes that are all broken :(

I think this should just mean enabling the appropriate markdown extension in whatever produces the site.

**Comment by rphair** [comment]
Another thing that glaringly doesn't work on the derived web site: cross-CIP links (https://github.com/cardano-foundation/CIPs/issues/109) which really hurts.  Still we are currently bound by https://github.com/cardano-foundation/CIPs/issues/436#issuecomment-1385303217 before we can talk about where & when this will be fixed (cc @KtorZ).

**Comment by KtorZ** [comment]
> I think this should just mean enabling the appropriate markdown extension in whatever produces the site.

Yes. @michaelpj be aware that no one likes the current website rendering. It's buggy and has poor rendering capabilities. This is on the TODO list for a while. This will likely be tackled as part of a common effort with developer portal (which also display CIPs in a rendered version). We may very likely just piggyback on their solution. 

Nothing is complicated here, it's just that no one has a couple of days to spend on those problems at this stage. We may start by formulating an update plan clearly maybe and call for help.

**Comment by michaelpj** [comment]
Personally, even if the CIPs are going to be incorporated elsewhere, a simple static site that just contained them and could be built and published in the repository CI seems nice to me. But you know what you're doing and I'm sure you'll come up with something good :)

**Comment by rphair** [comment]
last cross-reference was from a user cited example of footnotes not working: https://github.com/cardano-foundation/CIPs/pull/428#issuecomment-1399582310

**Comment by rphair** [comment]
@KtorZ @michaelpj this has been fixed on `cips.cardano.org` by one of these PRs submitted — somewhere in this time range, after the CIP editors became more regularly involved in the testing of & feedback for that site:
* https://github.com/cardano-foundation/cf-cip-frontend/pull/54
* https://github.com/cardano-foundation/cf-cip-frontend/pull/56
* https://github.com/cardano-foundation/cf-cip-frontend/pull/57

BTW if footnotes were glitchy in rendering on GitHub itself, whatever problems we had in 2023 have vanished by now.  For verification that footnotes are in order today on both platforms please see CIP-0140 which makes more use of footnotes than any other CIP/CPS so far.
